test/fixtures/transformation/es6-tail-call/try-catch/expected.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+(function f(n) {
+  if (n <= 0) {
+    return "foo";
+  }
+  try {
+    return f(n - 1);
+  } catch (e) {}
+})(1000000) === "foo";
+
+(function f() {

```

Comment:
That would duplicate destructuring etc and would cause defaults to be evaluated twice. ie:

```javascript
function foo(bar = foobar()) {

}

foo();
```